src/Aspire.Hosting.Dapr/DaprDistributedApplicationLifecycleHook.cs

```diff
@@ -45,6 +40,10 @@ public async Task BeforeStartAsync(DistributedApplicationModel appModel, Cancell
 
         var sideCars = new List<ExecutableResource>();
 
+        var fileName = this._options.DaprPath
+            ?? GetDefaultDaprPath()
+            ?? throw new DistributedApplicationException("Unable to locate Dapr executable.");
```

Comment:
Nit: I might use "Unable to locate the Dapr CLI." which could help differentiate between it and the Dapr runtime (i.e. `daprd`).  The two are sometimes confused.